docs/data/osdf/install-origin-rpm.md

```diff
@@ -0,0 +1,172 @@
+title: Installing the OSDF Origin by RPM
+
+Installing the OSDF Origin
+==========================
+
+This document describes how to install a Pelican-based Open Science Data Federation (OSDF) Origin service via RPMs.
+This service allows an organization to export its data to the Data Federation.
+
+!!! note
+    The origin must be registered with the OSG prior to joining the data federation.
+    You may start the registration process prior to finishing the installation by [using this link](#registering-the-origin) 
+    along with information like:
+
+    * Resource name and hostname
+    * VO associated with this origin server (which will be used to determine the origin's namespace prefix)
+    * Administrative and security contact(s)
+    * Who (or what) will be allowed to access the VO's data
+    * Which caches will be allowed to cache the VO data
+
+
+Before Starting
+---------------
+
+Before starting the installation process, consider the following requirements:
+
+* __Operating system:__ A RHEL 8 or RHEL 9 or compatible operating systems.
+* __User IDs:__ If they do not exist already, the installation will create the Linux user ID `xrootd` for running daemons.
+* __Host certificate:__ Required for authentication.  See note below.
+* __Network ports:__ The origin service requires the following ports open:
+  * Inbound TCP port 8443 for file access via the HTTP(S) and XRoot protocols.
+  * (Optional) Inbound TCP port 8444 for access to the web interface for monitoring and configuration;
+    if enabled, this should be restricted to the LAN or management network.
```

Comment:
```suggestion
    if enabled, consider restricting access from your LAN.
```

Management network implies BMC / IPMI

Reply:
Copied the language from the cache doc, which doesn't have 'management network'.